fhir-server-test/src/test/java/com/ibm/fhir/server/test/BundleTest.java

```diff
@@ -829,14 +827,12 @@ public void testBatchSearchWithEncoding() throws Exception {
         
         Bundle responseBundle = getEntityWithExtraWork(response,method);
         
-        assertResponseBundle(responseBundle, BundleType.BATCH_RESPONSE, 2);
+        assertResponseBundle(responseBundle, BundleType.BATCH_RESPONSE, 1);
```

Comment:
Why is this 1?  Is it suppose to be 1? 

I got sucked into this before where the code was wrong.

Reply:
I removed the entry from lines 820 and 821, so now we expect 1 instead of 2.

Specifically, now that we're strict by default this second request was failing due to the invalid "Jeter" parameter name.  I don't think that was needed for the test (which is ensuring that the `%26` gets properly unencoded) and so I removed it.

Reply:
as an Annoying RedSox fan... this makes me happy

** and double checking the intent